src/Card/CardGrid.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useMemo } from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+
+const CardGrid = ({
+  className,
+  children,
+  columnSizes,
+}) => {
+  const cards = useMemo(
+    () => React.Children.map(children, card => (
+      <Col {...columnSizes}>
+        {card}
+      </Col>
+    )),
+    [children, columnSizes],
+  );
+
+  return (
+    <div className={classNames('pgn__card-grid', className)}>
+      <Row>
+        {cards}
+      </Row>
+    </div>
+  );
+};
+
+CardGrid.propTypes = {
+  /** The class name for the CardGrid component */
+  className: PropTypes.string,
+  /** The Card components to organize into a responsive grid */
+  children: PropTypes.node.isRequired,
+  /** An object containing the desired column size at each breakpoint */
+  columnSizes: PropTypes.shape({
+    xs: PropTypes.number,
+    sm: PropTypes.number,
+    md: PropTypes.number,
+    lg: PropTypes.number,
+    xl: PropTypes.number,
+  }),
```

Comment:
Does this map to the `Col` prop api?

Reply:
It does! Technically, you could pass in whatever props here `Col` accepts.